### PR TITLE
Change sbix test font unitsPerEm to 800, glyf table line contour

### PR DESCRIPTION
**Pull request by drott**
Change the unitsPerEm to 800, as versions of Mac OS older than 10.13
have problems in CoreText determining the glyph bounds of bitmap glyphs
with unitsPerEm set to 1024 or numbers not divisible by 100 when the
contour is not closed. This is needed in combination with the change
below:

Rendering the sbix test font on Mac OS 10.12 can lead to seeing a black
square instead of a red one due to the sbix bitmap being overpainted
with the glyf contour, compare [1]. To avoid this, change the glyph
contour to a degenerate line from 0,0 to 800,800.

[1] https://bugs.webkit.org/show_bug.cgi?id=170124

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2016 Roel Nieskens, https://pixelambacht.nl
+Copyright (c) 2018 Google LLC
```
Comment (RoelN) [inline LICENSE.md]: Why is Google getting the copyright here? :-)

Comment (drott) [reply LICENSE.md]: Thanks for the review. When contributing to open source projects, we're asked to add Google as an author.

**Comment by RoelN** [comment]
Many thanks for the contribution! I didn't know about this CoreTex bug, so thanks for bringing it to my attentition and taking the time to create a PR!

**Comment by drott** [comment]
Nice, thanks for the quick merge.

**Pull request merged**